package.json

```diff
@@ -1,25 +1,32 @@
 {
   "name": "@novnc/novnc",
-  "version": "1.6.0",
+  "version": "1.6.1-alpha.1",
   "description": "An HTML5 VNC client",
-  "browser": "lib/rfb",
```

Comment:
Shouldn't this be kept in some form to indicate that this is code meant to be run in a browser?

Reply:
I don't know which software reads this in the presence of "exports" I could not find any examples

Reply:
Reading the npm documentation, this isn't entirely clear. It states that "exports" replaces "main". And since "browser" is an alternative to "main", I suppose we can assume that it is also replaced.

So let's go with your suggestion.

It would be nice if there was something in "exports" that still indicates that this is a browser package, but I can't find anything along those lines.